style.css

```diff
@@ -0,0 +1,130 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+html {
+  width: 100%;
+  height: 100%;
+}
+
+body {
+  width: 100%;
+  height: 100%;
+  font-size: 18px;
+}
+
+main {
+  width: 100%;
+  height: 100%;
+}
+
+.container {
+  width: 100%;
+  max-width: 450px;
+}
+
+header {
+  background-color: #3c3a39;
+  width: 100%;
+  height: 5rem;
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+  padding: 1rem;
+}
+
+.menu {
+  position: static;
+  width: auto;
+  height: auto;
+}
+
+.logo-container p {
+  color: white;
+  font-size: 18px;
+  line-height: 24px;
+  font-weight: 700;
+  font-family: "Roboto", serif;
+}
+
+.time-style {
+  position: absolute;
+  width: 66.66px;
+  height: 50.34px;
+}
+
+.Group {
+  position: calc(23);
+  width: 66px;
+  height: 50px;
+}
+
+.home {
+  background-color: #1c1a19;
+  height: 100vh;
+  padding: 5rem 0 1rem 1rem;
```

Comment:
Kindly, I suggest you to use flexbox to center elements to the container, this will help your site to be more adaptable to any size of device.

Reply:
@mgmediaweb
Thanks for your good and valuable feedback.